processing/src/main/java/org/apache/carbondata/processing/newflow/AbstractDataLoadProcessorStep.java

```diff
@@ -114,11 +114,15 @@ protected CarbonRowBatch processRowBatch(CarbonRowBatch rowBatch) {
   /**
    * It is called when task is called successfully.
    */
-  public abstract void finish();
+  public void finish() {
+    // implementation classes can override to update the status.
+  }
 
   /**
    * Closing of resources after step execution can be done here.
    */
-  public abstract void close();
+  public void close() {
+    // implementation classes can override to close the resources if any available.
```

Comment:
Removed `finish` method and kept only `close`, so for all cases `close` need to be called.